swagger.yml

```diff
@@ -1014,6 +1014,34 @@ paths:
             $ref: '#/definitions/plugin'
         default:
           $ref: '#/responses/error'
+  /sites/{site_id}/plugin_runs/latest:
+    get:
+      operationId: getLatestPluginRuns
+      tags: [x-internal]
+      description: 'This is an internal-only endpoint.'
+      parameters:
+        - name: site_id
+          in: path
+          type: string
+          required: true
+        - name: packages
+          in: query
+          type: array
+          items:
+            type: string
+          required: true
+        - name: state
+          in: query
+          type: string
+      responses:
+        '200':
+          description: OK
+          schema:
+            type: array
+            items:
+              $ref: '#/definitions/pluginRun'
```

Comment:
Does the API modify the `plugin_run` when storing it?
Or does it mostly match what `@netlify/build` sent when performing `createPluginRun`?

Reply:
We're not currently doing any modification on the backend, it will be returned with the same values that were sent by `@netlify/build`.

Reply:
Great, this should make it easier to assume its shape :+1: